shell/packages/sandstorm-ui-topbar/topbar.html

```diff
@@ -51,6 +51,17 @@
           <button class="close-button" title="Close {{title}}"></button>
         </li>
         {{/each}}
+
+        {{#with accountExpires}}
+          <li><div class="demo-notice">
+            <p>This demo expires in: <span class="countdown {{#if urgent}}urgent{{/if}}">{{countdown}}</span></p>
+
+            <p>To make your data permanent, create an account:
+              <button class="sign-in">Create account »</button></p>
```

Comment:
Ah hah, here's the lowercase one.

It'd be cool to be consistent about "Create Account" (capital A) vs. "Create account", unless there's a reason not to.